Plugin/OrderRepositoryPlugin.php

```diff
@@ -20,11 +21,26 @@ public function __construct(OrderPaymentExtensionFactory $orderPaymentExtensionF
      * and add the additional information as a json encoded string to an extension attribute.
      */
     public function afterGet(OrderRepositoryInterface $orderRepository, OrderInterface $order)
+    {
+        $this->changeAdditionalInformation($order);
+        return $order;
+    }
+
+    public function afterGetList(OrderRepositoryInterface $orderRepository, OrderSearchResultInterface $orders)
```

Comment:
this is a minor detail, `$orderRepository ` can be renamed to `$subject` - we follow this convention when writing plugin methods